### PR TITLE
🐛 More needed for podman workaround in kind-load

### PR DESCRIPTION
**Pull request by bentito**
# Description

The commit adding the podman workaround for the kind-load target seems to need a bit more work:

It needs to have the tag name set before image-archive is saved 

## Reviewer Checklist

- [ ] API Go Documentation
- [ ] Tests: Unit Tests (and E2E Tests, if appropriate)
- [x] Comprehensive Commit Messages
- [ ] Links to related GitHub Issue(s)

### CODE REVIEW

**Comment by trgeiger** [comment]
Oh nice, I was just about to put up something for this.

**Comment by trgeiger** [comment]
note: the changes here and in #790 were required for me to successfully run`make run` on my Fedora machine.

File: Makefile [hunk 1]
```diff
@@ -149,13 +149,16 @@ extension-developer-e2e: run image-registry test-ext-dev-e2e kind-clean
 e2e-coverage:
 	COVERAGE_OUTPUT=./e2e-cover.out ./hack/e2e-coverage.sh
 
-.PHONY: kind-load
```
Comment (grokspawn) [inline Makefile:152]: Why remove the .PHONY target annotation here?  We don't generate a versioned artifact as a result of this make target.

Comment (trgeiger) [reply Makefile:152]: I believe this was an accidental keystroke, will fix and thanks for catching.

Comment (bentito) [reply Makefile:154]: ugh, I just blew up this PR with an ill-advised force push. For a small change like this I'm going to start over.

**Comment by bentito** [comment]
I'm going to try this again from clean upstream:main

**Pull request closed without merging**